dGame/dComponents/LevelProgressionComponent.cpp

```diff
@@ -7,6 +7,8 @@
 LevelProgressionComponent::LevelProgressionComponent(Entity* parent) : Component(parent) {
 	m_Parent = parent;
 	m_Level = 1;
+	m_SpeedBase = 500.0f;
+	m_CharacterVersion = 1;
```

Comment:
What are your thoughts on making an enum for character versions so that instead of these being arbitrary values, the versions would have some kind of value associated with them?

Reply:
That sounds good, can discuss this more off GH